### DISCUSSION TITLE
Heads-up: dropping support for USD 19.11

### DISCUSSION THREAD

**Discussion by kxl-adsk** [Autodesk/maya-usd#1101] [Announcement]
USD and Hydra continue to evolve rapidly and several contributors have called out the cost of maintaining support for older versions of USD. To address this, our Technical Steering Committee has decided to limit support of USD versions to 1 year.

With the release of USD 21.02 this week (and already supported by MayaUSD), we will be dropping support for USD 19.11 soon.

**Comment by mattyjams** [comment]
I just posted #1138, which contains the changes that would put this into effect.
